src/menu/settings.c

```diff
@@ -18,6 +18,7 @@ static settings_t init = {
     .bgm_enabled = false,
     .sound_enabled = false,
     .rumble_enabled = true,
+    .auto_firmware_update_enabled = false,
```

Comment:
All "official" ED64 OS just try to load latest firmware into the cart. Do we really need a switch for that?

Reply:
Also I would strongly advise to not store flashcart specific settings in this file. I would rather use `.ini` file that is created in the `ed64_state.c` to store this switch.

Reply:
Done.